bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/config/config.xml

```diff
@@ -103,6 +103,18 @@
 		</parameter>
 	</config-description>
 
+	<config-description uri="netatmo:weather">
+		<parameter name="id" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})" required="true">
+			<label>@text/config.equipmentId.label</label>
+			<description>@text/config.equipmentId.description</description>
+		</parameter>
+		<parameter name="refreshInterval" type="integer" min="20" unit="s">
+			<label>@text/config.refreshInterval.label</label>
+			<description>@text/config.refreshInterval.description</description>
+			<default>600</default>
```

Comment:
Given that this is not a parameter that should usually be tangled with, this would be worth considering:
```suggestion
			<default>600</default>
			<advanced>true</advanced>
```

Reply:
Added